### PR TITLE
feat: adding basic colors, fonts, styles (WIP)

### LINKED ISSUE #1: Port the generic styles for the CSS
- corporate colors
- buttons
- default margins/paddings
- media query breakpoints
- fonts

### PR DESCRIPTION
**Pull request by solaris007**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #1

Test URLs:
- Before: https://main--rolex-event--hlxsites.hlx.live/
- After: https://issue-1--rolex-event--hlxsites.hlx.live/

### CODE REVIEW

File: styles/lazy-styles.css [hunk 1]
```diff
@@ -1 +1,52 @@
-/* below the fold CSS goes here */
\ No newline at end of file
+/* below the fold CSS goes here */
+
+@font-face {
```
Comment (ramboz) [inline styles/lazy-styles.css:3]: Looking at https://issue-1--rolex-event--hlxsites.hlx.page/, the async fonts are quite noticeable.
You definitely see the font flash, and we do create some CLS it seems.

File: styles/styles.css [hunk 2]
```diff
@@ -10,239 +10,366 @@
...
+    --device-laptops-min-width: 600px;
+    --device-desktops-s-min-width: 768px;
+    --device-desktops-m-min-width: 1024px;
+    --device-desktops-l-min-width: 1600px;
+
+    /* colors */
+    --background-color: #fff;
+    --button-background-color: transparent;
+    --button-border-color: var(--link-color);
+    --button-color: var(--link-color);
```
Comment (ramboz) [inline styles/styles.css:25]: I think we could define a few:
```
--primary-color:
--secondary-color:
…
```

so it's more verbose, and we don't use a "link" color on other elements. In that sense the boilerplate isn't "academically" clean to me

File: styles/styles.css [hunk 3]
```diff
@@ -10,239 +10,366 @@
...
 
 body {
-  font-size: var(--body-font-size-m);
-  margin: 0;
-  font-family: var(--body-font-family);
-  line-height: 1.6;
-  color: var(--text-color);
-  background-color: var(--background-color);
-  display: none;
+    background-color: var(--background-color);
```
Comment (ramboz) [inline styles/styles.css:94]: It would default to white already, so you can technically skip it

File: styles/styles.css [hunk 4]
```diff
@@ -10,239 +10,366 @@
...
-  --heading-font-size-xl: 40px;
-  --heading-font-size-l: 32px;
-  --heading-font-size-m: 24px;
-  --heading-font-size-s: 20px;
-  --heading-font-size-xs: 18px;
-
-  /* nav height */
-  --nav-height: 64px;
+:root {
+    /* device minimum widths */
```
Comment (ramboz) [inline styles/styles.css:14]: Not sure if it is intentional, but you switched to 4-spaces indentation from the initial 2-spaces

File: styles/styles.css [hunk 5]
```diff
@@ -10,239 +10,366 @@
...
+}
+
+a.button.secondary:hover, a.button.secondary:focus {
+    background-color: var(--button-secondary-hover-background-color);
+    border-color: var(--button-secondary-hover-border-color);
+    cursor: pointer;
+}
 
 
 main input {
```
Comment (ramboz) [inline styles/styles.css:258]: I'd be tempted to comment those out until we actually see an input field being used

File: styles/styles.css [hunk 6]
```diff
@@ -10,239 +10,366 @@
...
-  border: 1px solid var(--text-color);
+    border: 1px solid var(--text-color);
 }
 
 main .section {
-  padding: 0 16px;
+    padding: 0 16px;
 }
 
 main pre {
```
Comment (ramboz) [inline styles/styles.css:280]: Maybe same here for pre and blockquote

**Review by ramboz — COMMENTED** [review]
Looks good to me, left a few comments.

File: styles/lazy-styles.css (hunk 1 above)
Comment (solaris007) [reply styles/lazy-styles.css:3]: ok, moving over to `styles.css`

File: styles/styles.css [hunk 7]
```diff
@@ -10,46 +10,143 @@
  * governing permissions and limitations under the License.
  */
 
- :root {
-  /* colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
+@font-face {
+  font-family: RolexFont;
+  src: url("/fonts/RolexFont-Regular-WebS.woff") format("woff"),
```
Comment (ramboz) [inline styles/styles.css:15]: we could theoretically skip the woff fonts, and just use the woff2 ones

**Review by ramboz — APPROVED** [review]
LHS 100 🎉

**Pull request merged**